packages/web/src/components/gcds-button/gcds-button.tsx

```diff
@@ -67,6 +72,8 @@ export class GcdsButton {
 
     if (!values.includes(newValue)) {
       this.size = 'regular';
+    } else if (this.buttonRole === 'start') {
+      this.size = 'regular';
```

Comment:
Does this need to be an else if, they're assigning `this.size` to the same value?

Reply:
Good catch! Removed extra if else statement